src/jit/emitarm64.cpp

```diff
@@ -9188,6 +9358,31 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             dst += emitOutput_Instr(dst, code);
             break;
 
+        case IF_LS_2D: // LS_2D   .Q.............. xx.xssnnnnnttttt      Vt Rn
+            elemsize = optGetElemsize(id->idInsOpt());
+            code     = emitInsCode(ins, fmt);
+
+            code |= insEncodeVectorsize(id->idOpSize()); // Q
+            code |= 0x5000;                         // xxx          - We only support the one register variant right now
+            code |= insEncodeVLSElemsize(elemsize); // ss
+            code |= insEncodeReg_Rn(id->idReg2());  // nnnnn
+            code |= insEncodeReg_Vt(id->idReg1());  // ttttt
+
+            dst += emitOutput_Instr(dst, code);
+            break;
+
+        case IF_LS_2E: // LS_2E   .Q.............. xx.Sssnnnnnttttt      Vt[] Rn
+            elemsize = optGetElemsize(id->idInsOpt());
+            imm      = emitGetInsSC(id);
+            code     = emitInsCode(ins, fmt);
+
+            code |= insEncodeVLSIndex(elemsize, imm); // Q xx S ss
```

Comment:
For `LD1 (multiple structures)` how we encode the number of destination registers?

Reply:
Yeah, I actually have the same question as it's one of the open things I have to look at for the structure instructions.

Reply:
multiple and single structures indicate how many elements you are reading from memory.

`ld1 (multiple structures)` reads an entire vector
`ld1 (single structure)` reads a single element and inserts it into a vector

Both forms support doing this operation to 1, 2, 3, or 4 target registers and this PR doesn't enable that support.

Once methods that support that are exposed; I imagine we will use `insOpts` flags to encode how many target registers there are, based on the intrinsic method being targeted.

Reply:
> Once methods that support that are exposed; I imagine we will use insOpts flags to encode how many target registers there are, based on the intrinsic method being targeted.

Ah ok, so I didn't miss anything then. thanks :)